src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -35,9 +36,25 @@ import { saveProfilePicture } from 'src/utils/image'
 import Logger from 'src/utils/Logger'
 import { useAsyncKomenciReadiness } from 'src/verify/hooks'
 import { Statsig } from 'statsig-react-native'
+import { ADJECTIVES, NOUNS } from './constants'
 
 type Props = StackScreenProps<StackParamList, Screens.NameAndPicture>
 
+export const _chooseRandomWord = (wordList: string[], randomSeed?: string) => {
```

Comment:
I don't think `randomSeed` is ever used b/c you mock out `seedrandom` for all the unit tests.

Reply:
Yeah good point, I'll remove it.